src/components/pages/Login/Login.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useState } from 'react';
+import { Redirect } from 'react-router-dom';
+import {
+  loginHandler,
+  getToken
+} from '../../../handlers/authenticationHandlers';
+
+function Login(props) {
+  const [credentials, changeCredentials] = useState({
+    username: '',
+    password: ''
+  });
+
+  const [status, changeStatus] = useState({
+    loading: false,
+    error: false
+  });
+
+  const submitLogin = async event => {
+    event.preventDefault();
+    changeStatus({ ...status, loading: true });
+    try {
+      await loginHandler({
+        username: credentials.username,
+        password: credentials.password
+      });
+      changeStatus({ error: false, loading: false });
+      return <Redirect to="/" />;
+    } catch (err) {
+      changeStatus({ error: true, loading: false });
+    }
+  };
+
+  if (localStorage.getItem(getToken())) {
```

Comment:
Does this actually work? I would expect the getToken method to directly return the token instead of the identifier

Reply:
Yes!! Good eye, well spotted 👀 This is basically because I've been bringing your changes constantly haha, thanks, fixed!